libbeat/docs/queueconfig.asciidoc

```diff
@@ -9,9 +9,12 @@ queue is responsible for buffering and combining events into batches that can
 be consumed by the outputs. The outputs will use bulk operations to send a
 batch of events in one transaction.
 
-You can configure the type and behavior of the internal queue by setting
-options in the `queue` section of the +{beatname_lc}.yml+ config file. Only one
-queue type can be configured.
+You can configure the type and behavior of the internal queue by
+setting options in the `queue` section of the +{beatname_lc}.yml+
+config file or by setting options in the `queue` section of the
+output. Only one queue type can be configured.  If both the top level
```

Comment:
> If both the top level queue section and the output section are specified the output section
takes precedence.

This is no longer true now that it will fail validation.